### ISSUE TITLE
CORS middleware doesn't work

### ISSUE THREAD

**Issue by CodinCat** [kataras/iris#108]
I'm trying to write a simple api server, just follow the cors middleware document

```go
// ... 

iris.Use(cors.New(cors.Options{})) // also tried cors.DefaultCors(), still not work
iris.Get("/", myHandler)
iris.Listen(*port)
```

and use javascript to fetch the result, but I get the CORS error:
`No 'Access-Control-Allow-Origin' header is present on the requested resource.`

did I do anything wrong or miss something?

thanks.

**Comment by kataras** [comment]
@CodinCat  Hello, can you upgrade iris with `go get -u github.com/kataras/iris` and tell me if that works ?

**Comment by CodinCat** [comment]
oh wow, it works..., thanks!

also thanks for this fantastic project.

**Comment by kataras** [comment]
I thanks you for the report!

**Comment by CodinCat** [comment]
Hello, I'm trying to run it on my production server (Ubuntu), and it doesn't work again..

Same code, local version is working (windows),
I've also tried to copy my local old iris to the server but still don't work

Not sure where the problem is

here is the log of crs.Log
```
[IRIS] Serve: Actual request
[IRIS]   Actual request no headers added: missing origin
```

**Comment by CodinCat** [comment]
oh sorry, it does work.

just my problem I guess

**Comment by kataras** [comment]
Don't worry :)

**Comment by intoxicated** [comment]
Could you please things you've changed on history or somewhere else? it's really hard to follow up to newest version..

**Comment by kataras** [comment]
History.md has everything about new changes and I'm trying to update my comments also
